db/kvstore_impl.go

```diff
@@ -56,6 +58,26 @@ func (m *memKVStore) Put(namespace string, key, value []byte) error {
 	return nil
 }
 
+func (m *memKVStore) Insert(name []byte, key uint64, value []byte) error {
+	return ErrNotSupported
```

Comment:
Is relevant with chainservice refactoring?

Reply:
as mentioned in the description, it is used in unit test